src/compiler/emitter.ts

```diff
@@ -324,6 +324,7 @@ namespace ts {
                 case SyntaxKind.PrivateKeyword:
                 case SyntaxKind.ProtectedKeyword:
                 case SyntaxKind.PublicKeyword:
+                case SyntaxKind.OverrideKeyword:

```

Comment:
you also need to handle this in the declarationEmitter.ts, and please add declaration emit tests.